### DISCUSSION TITLE
响应器规则, 会使用到一个编辑器, 这个编辑器会引用外部js, 但是我项目是内网的, 这个怎么解决呢

### DISCUSSION THREAD

**Discussion by jisao** [alibaba/formily#4104] [Q&A 2.x]
所引用的外部服务是  https://unpkg.com/monaco-editor@0.30.1/min/vs/base/worker/workerMain.js#editorWorkerService

**Comment by zonefile** [comment]
打开链接后 `Ctrl+S` 保存到计算机本地，再上传到内网服务器，获取链接后替换掉原来这个。
